cypress/integration/contact/contact.test.js

```diff
@@ -0,0 +1,60 @@
+/// <reference types="cypress" />
```

Comment:
- Why you don't mock responses from server? We don't have any contacts in our test database.

- Why you don't cover add, edit, validation and delete comment functionality.

Reply:
Please change naming of values data-cy attibutes to kebab case.